### PR TITLE
Add Ooyala Pulse plugin v4.3.3

### PR DESCRIPTION
**Pull request by eshaglund**
Fix for PSDK-366: HTML5 Player on Android showing black screen when playing content.

### CODE REVIEW

File: js/pulse.js [hunk 1]
```diff
@@ -806,7 +806,8 @@
          */
         this.sharedElementGive = function() {
             setTimeout(function(){
-                _adManager.ui.ooyalaVideoElement[0].play();
+                _adManager.sharedVideoElement.style.display = "block";
+                _adManager.sharedVideoElement.play();
             }, 100);
             _adManager.sharedVideoElement.style.visibility ="hidden";
```
Comment (angieisrani) [inline js/pulse.js]: @pilievOoyala I think the plugin shouldn't need to set visibility since vtc is doing that - are you able to verify this?

Comment (pilievOoyala) [inline js/pulse.js]: Correct, we shouldn't need to edit the visibility.  Have you tried debugging in to the video controller and seeing if there is a bug there?  If this plugin edits the visibility and the VTC does, then we can get out of sync. Which is what I suspect may be the root cause.

Comment (pilievOoyala) [inline js/pulse.js]: Also, in the future please feel free to remind us to review these changes so they don't get forgotten on our side.

**Comment by eshaglund** [comment]
Changes merged in another PR

**Pull request closed without merging**